recipes/phyfleaux/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2020 R. Tohid
+#
+# Distributed under the Boost Software License, Version 1.0. (See a copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+FROM ubuntu:18.04
+
+RUN apt update
+RUN apt install -y sudo
+
+# add user
+RUN adduser --disabled-password --gecos '' stellar
+RUN adduser stellar sudo
```

Comment:
These commands seem to appear about 4 times in this file. Why?

Reply:
@stevenrbrandt can you please point to the other places, I cannot find them.

Reply:
It was replicated yesterday, but not today :-P

Reply:
It's repeated in all the files 😨 ... I'll get rid of all these scripts once [builder](https://github.com/STEllAR-GROUP/docker-containers/blob/master/image_builder.py) is stable.

By the way, I am thinking to move to [singularity](https://singularityhub.github.io/singularity-cli/install) image builder, what do you think @stevenrbrandt?

Reply:
I would stick with docker. You can build singularity images from docker images, but not vice-versa so far as I know.